hash/super_tv_pc_cart.xml

```diff
@@ -71,7 +67,16 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<!-- C-06 -->
+	<software name="doraepp" supported="no">
+		<description>Doraemon Excite Ping-Pong</description>
+		<year>2006</year>
+		<publisher>Epoch</publisher>
+		<part name="cart" interface="super_tv_pc_cart">
+			<dataarea name="prg" size="0x200000">
+				<rom name="c-06.u1" size="0x200000" crc="bdf25763" sha1="f3f20e8476d086d26152cdd46164b4cad736e3ed" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
I pasted in the thing that was removed, but since that wasn't the string that actually gave me any hits when buying it, it could just be complete nonsense for all I know